Updated the README with all the new groups info (#34)
AssemblyScript/working-group: README.md

For this, we will be following a meritocracy. Where consensus will be requested on issues opened on this repo to get a community vote, and can find a path going forward.

# Groups

The AssemblyScript project has meetings and and correspondance divided into groups. These groups communicate and work together very closely, and provides a way for focused discussions for those who need that.

## Working Group

The working group is for those who would like to contribute to the project directly. Whether that be through getting advice on technical implementations, showing off a new tool they are building, or just generally talking about the AssemblyScript project and where it is headed.

**Anyone working on anything related to the AssemblyScript project is welcome to participate in the Working Group!**

## Communitty Group

> The [community group](https://github.com/AssemblyScript/community-group) is for those who would like to discuss with other communitty members (including members of the working group) using AssemblyScript in their applications. The goal being that they can prioritize their core feature requests and use cases and communicate that to the working group. Which can help reduce duplicating efforts amongst the communitty, and allow the communitty to be collectively more productive!

Reviewer: nit: communitty -> community

Author: Oh yeah I can't spell :joy: I just saw @dcodeIO opened #35 

So we're going to figure out how to merge the two PRs :upside_down_face:

> The [community group](https://github.com/AssemblyScript/community-group) is for those who would like to discuss with other communitty members (including members of the working group) using AssemblyScript in their applications. The goal being that they can prioritize their core feature requests and use cases and communicate that to the working group. Which can help reduce duplicating efforts amongst the communitty, and allow the communitty to be collectively more productive!

Reviewer 2: ```suggestion
The [community group](https://github.com/AssemblyScript/community-group) is for those who would like to discuss with other community members (including members of the working group) using AssemblyScript in their applications. Its goal is to communicate core feature requests and use cases to aid in prioritizing the working group's efforts.  Which can help reduce duplicating efforts amongst the community, and allow the community to be collectively more productive!
```

Author: reflected in #35 :smile: :+1:
